src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -14,6 +14,7 @@
 public class FeedbackRubricQuestionDetails extends FeedbackQuestionDetails {
     public int numOfRubricChoices;
     public List<String> rubricChoices;
+    public List<Double> rubricWeights;
```

Comment:
Have you tested with legacy data? Will there be problems with old rubrics questions without <code>rubricsWeights</code>?